index.d.ts

```diff
@@ -52,8 +52,26 @@ declare module 'fastify' {
       [statusCode: string]: OpenAPIV3.ResponseObject['links'];
     }
   }
+  
+  interface FastifyContextConfig {
+    swaggerTransform?: SwaggerTransform | null;
+  }
 }
 
+type SwaggerTransform = <S extends FastifySchema = FastifySchema>({
```

Comment:
Why not
```typescript
type SwaggerTransform<S extends FastifySchema = FastifySchema> = ({
```

And is it still possible to specify the generic?

Reply:
I simply extracted the type from the global transform so I could re-use the same function type. I think it is correct; the function should be generic, not the type.

Reply:
Thanks for the clarification. Lets see if somebody complains... :))